batch/batch-runner.sh

```
@@ -472,39 +429,25 @@ process_offer() {
 
-  # Launch claude -p worker.
-  # Model defaults to the Claude Max subscription default unless --model was
-  # passed. Building the command in an array keeps quoting safe regardless.
-  # --strict-mcp-config (with no --mcp-config) starts workers with no MCP
-  # servers: they only evaluate offers and need none. Without it each parallel
-  # worker inherits the parent session's MCP (e.g. Playwright) and they deadlock
-  # fighting over the single shared browser when --parallel > 1 (issue #506).
-  local -a claude_args=(-p --dangerously-skip-permissions --strict-mcp-config)
+  # Launch worker via AGENT_CMD.
+  # Model defaults to the CLI's default unless --model was passed.
+  local combined_prompt=$(cat "$resolved_prompt" && echo "" && echo "$prompt")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)